PhysicsTools/NanoAOD/python/photons_cff.py

```diff
@@ -207,12 +212,24 @@ def make_bitmapVID_docstring(id_modules_working_points_pset):
         mvaID_Fall17V1p1 = Var("userFloat('mvaID_Fall17V1p1')",float,doc="MVA ID score, Fall17V1p1",precision=10),
         mvaID_WP90 = Var("userInt('mvaID_WP90')",bool,doc="MVA ID WP90, Fall17V2"),
         mvaID_WP80 = Var("userInt('mvaID_WP80')",bool,doc="MVA ID WP80, Fall17V2"),
+        pfPhoIso03 = Var("photonIso()",float,doc="PF absolute isolation dR=0.3, photon component (uncorrected)"),
+        trkSumPtHollowConeDR03 = Var("trkSumPtHollowConeDR03()",float,doc="Sum of track pT in a hollow cone of outer radius, inner radius"),
+        chargedHadronIso = Var("chargedHadronIso()",float,doc="charged hadron isolation with dxy,dz match to pv"),
+        pfChargedIsoPFPV = Var("chargedHadronPFPVIso()",float,doc="PF absolute isolation dR=0.3, charged component (PF PV only)"),
```

Comment:
The two variables were are probably fully redundant, we should verify and only have one.

Reply:
so based on the discussion in the mattermost group can I remove ```pfChargedIsoPFPV```?